go/nogo.rst

```diff
@@ -262,11 +262,14 @@ default).
         visibility = ["//visibility:public"],
     )
 
-Setting ``vet = True`` is equivalent to adding the ``atomic``, ``bool``,
-``buildtags``, ``nilfunc``, and ``printf`` analyzers from
+Setting ``vet = True`` is equivalent to adding the ``atomic``, ``bools``,
+``buildtag``, ``nilfunc``, and ``printf`` analyzers from
 ``@org_golang_x_tools//go/analysis/passes`` to the ``deps`` list of your
 ``nogo`` rule.
 
+See the full list of available nogo checks:
+``bazel query 'kind(go_tool_library, @org_golang_x_tools//go/analysis/passes/...)'``
```

Comment:
I think it would be better to pull this into a code block. Right now, my browser wraps the command to another line, which makes it hard to read.